docs/framework/theming.md

```diff
@@ -62,8 +62,8 @@ Most components have props that define their look and feel. Most common are the
   [MUI Button documentation ↗](https://mui.com/components/buttons/)
 - To learn which options are accepted by a prop, refer to the component's API:
   [MUI Button API ↗](https://mui.com/api/button/). You can also use your IDE's
-  suggestions functionality. For [VS Code's](../getting-started/vscode.md)
-  IntelliSense feature, type Ctrl+Space.
+  suggestions functionality. For [VS Code's](./vscode.md) IntelliSense feature,
```

Comment:
Did you check if this change works? Because I believe there is a bug in the documentation website that creates the wrong url for `./` links.

Reply:
Fixed!